apps/bfd-server/bfd-server-war/src/main/resources/security_labels.csv

```diff
@@ -0,0 +1,678 @@
+system,code,startDate,endDate
+http://www.ama-assn.org/go/cpt,4320F,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0005,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0006,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0007,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0008,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0009,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0010,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0011,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0012,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0013,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0014,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0015,1970-01-01,ACTIVE
+https://www.cms.gov/Medicare/Coding/HCPCSReleaseCodeSets,H0050,1970-01-01,ACTIVE
+http://www.ama-assn.org/go/cpt,99408,1970-01-01,ACTIVE
+http://www.ama-assn.org/go/cpt,99409,1970-01-01,ACTIVE
```

Comment:
Not a big deal, but for ease of reading through this, maybe it's better to have all the codes grouped by system?